### ISSUE TITLE
Header no longer has rounded corners (regression)

### ISSUE THREAD

**Issue by romaninsh** [fomantic/Fomantic-UI#308]
# Bug Report

After switching our project from Semantic to Fomantic, we noticed that dialog top edges are no longer rounded. After some digging we noticed that "header" had rounded corners before, but it is no longer the case:

``` css
/*******************************
             Modal
*******************************/

.ui.modal > :first-child:not(.icon),
.ui.modal > .icon:first-child + * {
  border-top-left-radius: 0.28571429rem;
  border-top-right-radius: 0.28571429rem;
}
```

Image before:
<img width="1595" alt="image" src="https://user-images.githubusercontent.com/453929/50095215-e5d02f80-020c-11e9-9ac5-5199d50dc4d0.png">

And after:
<img width="1605" alt="image" src="https://user-images.githubusercontent.com/453929/50095229-f1bbf180-020c-11e9-898e-37ee181db246.png">

This affects all the dialogs.

## Steps to reproduce
1. TBA
2.
3.

## Testcase
http://ui.agiletoolkit.org/demos/modal2.php

## Version
2.6.4

**Comment by romaninsh** [comment]
Looks like it's caused by "ui inverted dimmer" which is inserted as a first div inside a modal, and therefore breaks selector:

<img width="383" alt="image" src="https://user-images.githubusercontent.com/453929/50095397-6000b400-020d-11e9-930c-2fbd23b0a982.png">

https://github.com/fomantic/Fomantic-UI/blob/master/src/definitions/modules/modal.less#L43

Perhaps `.ui.modal > :first-child:not(.icon,.dimmer)` could fix it?

**Comment by romaninsh** [comment]
Possibly that's a bug of atkjs, created related issue there.

**Comment by hammy2899** [comment]
@romaninsh I reckon it is https://fomantic-ui.com/modules/modal.html#/examples all those examples seem fine

**Comment by lubber-de** [comment]
Please adjust your (backend/atk)-code to not insert an additional manual dimmer div when working with modals.
Creating the Dimmer-tag for modals is handled by the module itself and is not positioned in the dom as shown.

**Comment by romaninsh** [comment]
There is no example with a closing "X" and dimmer.

**Comment by lubber-de** [comment]
> There is no example with a closing "X" and dimmer.

:confused: A modal always creates an internal dimmer and an example having a close icon can be found here https://fomantic-ui.com/modules/modal.html#full-screen

The created dom has the close icon as its first div child and the dimmer itsel is surrounded

![image](https://user-images.githubusercontent.com/18379884/50097108-70b72700-0219-11e9-953c-74c7a67ed88c.png)

**Comment by lubber-de** [comment]
I found it. it's used in the `innerDimmer` method of `modal.js` which creates the dimmer div you mentioned when `allowMultiple` is set to true.Please try if setting this to false basically solves the issue.
In the meantime i'll prepare a fiddle to reproduce this

**Comment by lubber-de** [comment]
@romaninsh It's fixed in #310 and supposed to be part of the next release.
See http://jsfiddle.net/co3gaj8q/
Sorry for the misunderstanding :slightly_smiling_face:

**Comment by romaninsh** [comment]
@lubber-de no worries. My investigation was also missleading. Thanks for addressing it so promptly.
